### DISCUSSION TITLE
Docker action: using existing image

### DISCUSSION THREAD

**Discussion by vitek-urbanec** [community/community#26394] [Actions]
<p>hello! I’m not entirely clear on how/if I can use an existing docker image from a registry (for instance ECR) for my Github Action. My use case is that I have a docker builder image that I’d like to use.</p>
<p>Thanks!</p>

**Comment by meiswjn** [comment, marked as answer]
<p>Not sure if I understand your question correctly, but…<br>
Of course :slight_smile: Just use</p>
<pre><code class="lang-auto">FROM [aws_account_id].dkr.ecr.us-west-2.amazonaws.com/[yourimage]
</code></pre>
<p>Has to be modified with your own registry URL, obviously.</p>

**Comment by vitek-urbanec** [comment]
<p>oh right! :man_facepalming: didn’t think of that. I was thinking of referencing the image within the action.yml rather than the Dockerfile. Expanding on that, how does the orchestration work? Could the local image on the github runner be referenced in other steps without the need for rebuild? And thanks!</p>

**Comment by meiswjn** [comment]
<p>I would assume that it does, yeah. But haven’t tried anything in that direction yet :slight_smile:</p>
